.kokoro/generate-docs.sh

```diff
@@ -38,6 +38,8 @@ python3 -m pip install --user django==2.2 ipython
 
 # Store the contents of bucket log in a variable to reuse.
 python_bucket_items=$(gsutil ls "gs://docs-staging-v2/docfx-python*")
+# Store empty tarballs that did not produce any content to check later.
```

Comment:
Design question: why not do this in the plugin? This seems like an error we should be notified about?

Reply:
Filed https://github.com/googleapis/docuploader/issues/114 to handle the issue on docuploader as well.